app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class RecipesController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:index, :show]
+  before_action :set_recipe, only: [:show, :edit, :update, :destroy]
+
```

Comment:
Is it normal that you did not create the destroy and index method? (Index should be the first method to be created :) )

Reply:
next step, i actually had it since it was easy but then wanted to focus on the branch and overdid it :D everything is still working though

Reply:
ah, and additionally i thought (two days ago) i would have to change it either way with pundit etc. will fix it!

Reply:
done!

Reply:
(index + destroy)